micromamba/tests/test_create.py

```diff
@@ -143,6 +143,24 @@ def test_env_lockfile_different_install_after_create(tmp_home, tmp_root_prefix,
     helpers.install("-p", env_prefix, "-f", install_spec_file, "-y", "--json")
 
 
+# Only run this test on Linux, as it is the only platform where xeus-cling
+# (which is part of the environment) is available.
+# @pytest.mark.timeout(60)
```

Comment:
I do not currently understand why this timeouts on the CI, whereas my manual test takes 10 seconds locally.

Reply:
And how much time does it take?

Reply:
Let me check when running the Python test suite.

Reply:
Honestly, let's just ship?

Reply:
Why? The test here doesn't do things the right way since we need to check that it doesn't timeout.
If we skip this, I don't see the point of adding it.
And I would rather spend more time doing things the right way than just ship and then still have the same problems again later...

Reply:
It actually took 93.58s.

Reply:
Hum, and when it hangs (without the fix) how much time does it take (with dry run)?

Reply:
It hangs forever (several hours).

If I set the level to 0 (which entirely removes logs), it takes less than 20 seconds.

Reply:
Ok so let's set 100 for the timeout.

Reply:
Yes, let me just look at the value of `verbosity` before to know whether the value passed as `level` was 3 or something less as it could still introduce a performance regression.